tendrl/commons/objects/node/atoms/check_service_status/__init__.py

```diff
@@ -0,0 +1,142 @@
+                               "on server-node. Error: %s" % (
+                                   service_name,
+                                   service.error
+                               ))
+                    else:
+                        msg = ("Service %s is not running on a server-node, "
+                               "Please start it manually or check the log "
+                               "file to figure out the exact "
+                               "problem" % service_name)
+                    logger.log(
+                        "error",
+                        NS.get("publisher_id", None),
+                        {
+                            "message": msg
+                        },
+                        job_id=self.parameters['job_id'],
+                        flow_id=self.parameters['flow_id']
+                    )
+                    service_status = False
+        else:
+            # check neccessary service status in storage nodes
+            for service_name in TENDRL_SERVICES["storage_node"]:
+                service = get_service_status(service_name)
+                if not service.running:
+                    if len(service.error) > 0:
+                        msg = ("Failed to check status of %s "
+                               "on %s. Error: %s" % (
+                                   service_name,
+                                   NS.node_context.fqdn,
+                                   service.error
+                               ))
+                    else:
+                        msg = ("Service %s is not running on %s, Please "
+                               "start it manually or check the log file to "
+                               "figure out the exact problem" % (
+                                   service_name,
+                                   NS.node_context.fqdn
+                               ))
+                    logger.log(
+                        "error",
+                        NS.get("publisher_id", None),
+                        {
+                            "message": msg
+                        },
+                        job_id=self.parameters['job_id'],
+                        flow_id=self.parameters['flow_id']
+                    )
+                    service_status = False
+            node_list = self.parameters['Node[]']
+            if service_status and len(node_list) > 1:
+                # check monitoring integration is running on server
+                try:
+                    node_arr = etcd_utils.read(
+                        "/indexes/tags/tendrl/integration/monitoring"
+                    ).value
+                    node_id = eval(node_arr)[0]
+                    service = NS.tendrl.objects.Service(
+                        service="tendrl-monitoring-integration",
+                        node_id=node_id
+                    ).load()
+                    service.error = list(service.error)
```

Comment:
Check if this is required

Reply:
i have checked this, we don't need to add this check, because it is already in list type